tests/resources/VCH-Util.robot

```diff
@@ -27,11 +27,7 @@ Set Test Environment Variables
     ${status}  ${message}=  Run Keyword And Ignore Error  Environment Variable Should Be Set  TEST_DATACENTER
     Run Keyword If  '${status}' == 'FAIL'  Set Environment Variable  TEST_DATACENTER  ${SPACE}
 
-    @{URLs}=  Split String  %{TEST_URL_ARRAY}
-    ${len}=  Get Length  ${URLs}
-    ${IDX}=  Evaluate  %{DRONE_BUILD_NUMBER} \% ${len}
-
-    Set Environment Variable  TEST_URL  @{URLs}[${IDX}]
+    Set Environment Variable  TEST_URL  %{TEST_ESX_IP}
```

Comment:
This will actually break all the nightlies as the nimbus setups only set TEST_URL_ARRAY and don't currently set TEST_URL or TEST_ESX_IP.  I would actually not like this to be called TEST_ESX_IP as we are actively moving towards VC installs and it doesn't make sense for nightlies either.  Just TEST_URL seems appropriate, and we can change the nightlies to set the TEST_URL var.

Reply:
I thought about renaming it to something else but wasn't sure. I agree with you to rename it to `TEST_URL` instead. I will update that and also change all `TEST_URL_ARRAY` for nightlies to `TEST_URL`.